hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -129,6 +129,8 @@
   public static final String OZONE_DB_CHECKPOINT_REQUEST_FLUSH =
       "flushBeforeCheckpoint";
 
+  public static final String SEPARATE_RATIS_PORTS_AVAILABLE = "RatisPorts";
+
```

Comment:
Can we define features in a separate class? We could also define them as enums.

Reply:
Yes, I can move it to separate class.  I'd rather avoid enums, which are more type-safe, but worse for compatibility.

Reply:
Although not sure if it's worth extracting to another class at this time.

Reply:
I was suggesting separate class so that in future if new features are defined, it could all be in the same class for better accessibility.

Reply:
Thanks @lokeshj1703 for the suggestion.  Extracted to separate class.  Also changed to using version numbers instead of "features".  I think this will be simpler in the long run.  Features of the datanode can be mapped at runtime (eg. if version > 0, then we have this "separate ports" feature).